### PR TITLE
Revamp community documentation

### PR DESCRIPTION
**Pull request by rrrutledge**
* Update README to represent the shifting focus of the working group towards any learning segments on InnerSource (not just the original ones sponsored by PayPal).
* Update CONTRIBUTING with the overall flow of how to work with the community to bring a segment from idea to finished product.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,23 +50,26 @@ workbook/0n-section-name.asciidoc // Contains the part of the workbook matching
 ```
 
 After material is finished the scripts will be used to film videos.
-Videos, articles, and workbooks will be reviewed by O'Reilly's professional editors and then published at _learning.oreilly.com_ and also at _innersourcecommons.org_.
+Each video should be about 5min in length.
+Each article should be about a page long.
```
Comment (lenucksi) [inline README.md:54]: A4, Letter, piece of parchment, 4k screen on 10 point, cowhide?
I often use a metric of "xyz words/letters" to describe a length of an article independent of its rendering. 
We could use 1000-2000 letters as a starter.
```suggestion
Each article should be about 1000-2000 letters, i.e. about a page, long.
```

**Review by lenucksi — APPROVED** [review]
This is good stuff, I've added one note though, feel free to merge after resolving.

File: README.md (hunk 1 above)
Comment (rrrutledge) [reply README.md:54]: Even though people _could_ be confused, do you think that people _will_ be confused?

Comment (lenucksi) [reply README.md:54]: > Even though people _could_ be confused, do you think that people _will_ be confused?

Let me mention this lovely German invention I found out about today, the [Normseite](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fde.m.wikipedia.org%2Fwiki%2FNormseite). Obviously everything needs to be standardized into oblivion or else! :wink: 

On a more serious note: I'm fine either way. If someone has questions they can still ask.

Comment (rrrutledge) [reply README.md:54]: OK I'm going to leave out for now.

**Pull request merged**